.github/workflows/app-build-verify.yml

```diff
@@ -254,28 +257,30 @@ jobs:
     # alive.
     #
     # Only runs when target platform and format are explicitly Linux and AppImage.
-    - name: Build AppImage project
+    - name: Build AppImage Project
       if: >
         startsWith(inputs.runner-os, 'ubuntu')
         && contains(fromJSON('["Linux"]'), inputs.target-platform)
         && contains(fromJSON('["AppImage"]'), inputs.target-format)
+      working-directory: ${{ steps.create.outputs.project-path }}
       run: |
-        cd tests/apps/verify-${{ inputs.framework }}
-        briefcase create linux AppImage
+        briefcase create linux AppImage \
+          ${{ steps.output-format.outputs.template-override }} \
+          --config 'requires=["toga-gtk==0.3.1", "PyGobject<3.46.0"]'
```

Comment:
Yeah; good point. From going through all the templates today, AppImage is only one I saw with a special additional configuration requirement beyond specifying `template=../../`.

I think for where to declare a build config override like this....I'm leaning towards here in `app-build-verify`....otherwise, any repo that needed to invoke a build with special requirements would need to be aware of such requirements....I'll consider it more though.